### ISSUE TITLE
Accessibility from a user's perspective

### ISSUE THREAD

**Issue by kentcdodds** [javascriptair/site#151]
**Title:** Accessibility from a user's perspective

**Description:** We often talk about accessibility (a11y) as something we all should be doing because there are people out there who have disabilities that make it a challenge to use the websites many of us are used to every day. But we don't get too many opportunities to hear about accessibility from a disabled user's perspective. Let's have that conversation and hear first hand what the experience is like and what we can do to make life better for everyone.

**Potential guest(s):** I don't know. Please suggest anyone you feel would feel comfortable discussing their perspective as a blind or otherwise disabled person.

If you haven't heard of [JavaScript Air](http://javascriptair.com) before, it's a live broadcast podcast all about JavaScript. Shows happen every Wednesday at [12:00 Central Time](http://www.thetimezoneconverter.com/). For more information about being a guest, see [this Google Doc](https://docs.google.com/document/d/1Wlg5NxTkDX8k77TnCx4Rgm-nHQKr5pzwGKN81k90XKQ/edit?usp=sharing). Note: we also have and enforce a [Code of Conduct](https://github.com/javascriptair/CODE_OF_CONDUCT) for the show and guests have nothing but great things to say about their experience on the show.

If you've been listed as a potential guest above, please comment on this issue about whether you're interested in coming on to talk about the subject and your availability (or unavailability) for the available dates on [the schedule](https://github.com/javascriptair/site/blob/master/other/schedule.md).

If you're interested, I'll need a little information from you for the website. Episode planning happens on the GitHub repo issues. Please comment on your episode’s issue with the following information (if you’d prefer, you can email this info to [javascriptair@gmail.com](mailto:javascriptair@gmail.com)):
- Your name (as you want it on the site)
- An email address to share a Google Doc with you
- Your twitter handle

Thanks!

P.S. Please forgive me if the language in the description is disrespectful in any way. I definitely don't mean to be disrespectful. Feel free to kindly correct me and I'll be happy to update things. Thanks for your patience :)

**Comment by parhamdoustdar** [comment]
Hi,

My name is Parham Doustdar. I'm a completely blind back-end programmer. To know me more, you can visit my website at https://www.parhamdoustdar.com.

I'd love to discuss accessibility and how it affects me and other blind users on your podcast if you're interested. Let me know and I'd be glad to oblige!

**Comment by kentcdodds** [comment]
Thank you for being willing to chat @parhamdoustdar. I think this will be a very useful show for people. As mentioned, the show is at 12:00 CT. Which of the following Wednesdays work well for you?
- Oct 5th
- Oct 12th
- Oct 19th

**Comment by kentcdodds** [comment]
Hi @parhamdoustdar, any chance you could come on the show on the 12th at 12:00 CT?
